tailwind.config.js

```diff
@@ -165,6 +165,8 @@ module.exports = {
             'critical-lg': 'var(--ods-shadow-critical-lg)',
             'critical-xl': 'var(--ods-shadow-critical-xl)',
             'critical-2xl': 'var(--ods-shadow-critical-2xl)',
+
+            none: 'var(--ods-shadow-none)',
```

Comment:
Ha, the joys of not extending the theme.

Reply:
🙈